service/src/test/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/ListMapperTest.java

```diff
@@ -81,5 +85,6 @@ public void shouldMapList() {
         assertThat(code.getDisplay()).isEqualTo("Triage");
         assertThat(listResource.getEntry().size()).isEqualTo(1);
         assertThat(listResource.getIdElement().getValue()).isEqualTo(RANDOM_UUID);
+        assertThat(deviceRef.getReference()).isEqualTo(listResource.getSource().getReference());
```

Comment:
you're comparing null with null here what might be false positive
let's check Reference (class) equality, not String